src/schema/v2/fair.ts

```diff
@@ -160,6 +160,11 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
           return dateRange(start_at, end_at, "UTC", format)
         },
       },
+      featuredKeywords: {
+        type: new GraphQLList(GraphQLString),
```

Comment:
i'm assuming this could prob be a non-null list of non-null strings, nullability-wise - `[String!]!` (empty list is just fine)

Reply:
What are you doing in here? 🏖️ 

I pushed an update as you recommended.

Reply:
Coming back to this, is it _actually_ appropriate to make this non-null,  given that the property is only returned in `Fair`'s `:public` JSON? A list endpoint would return `:short` JSON where this field will be absent.

Reply:
If the Gravity response might be null then just fallback onto empty array

Reply:
🙏